File: 2_JAVA/Assignment 4/CurvedBanner.java
package Banner;
public class CurvedBanner extends Banner{
	public CurvedBanner(double x,double y,float r){
		super(x,y);
		radius = r;
	}
	public double area(){
		return super.area() - 0.86 * Math.pow(radius,2);
	}
	private float radius;
}

File: Assignment3/DemoApp/curvedbanner.cs
namespace Met.Banner;
public class CurvedBanner :Banner
{
	public CurvedBanner(double x,double y,float r) : base(x,y)
    {
	
		radius = r;
	}
	public override double Area()
    {
		return base.Area() - 0.86 * Math.Pow(radius,2);
	}
	private float radius;
}

